draft-ietf-mls-protocol.md

```diff
@@ -2447,26 +2436,48 @@ and administered under a Specification Required policy {{!RFC8126}}.
 
 ## MLS Ciphersuites
 
-The "MLS Ciphersuites" registry lists identifiers for suites of cryptographic
-algorithms defined for use with MLS.  These are two-byte values, so the maximum
-possible value is 0xFFFF = 65535.  Values in the range 0xF000 - 0xFFFF are
-reserved for vendor-internal usage.
+A cipher suite value defines a combinaison of a protocol version
+and of the set of cryptographic algorithms used for this version MLS.
 
-Template:
+Cipher suite names follow the naming convention:
 
-* Value: The two-byte identifier for the ciphersuite
-* Name: The name of the ciphersuite
-* Reference: Where this algorithm is defined
-
-The initial contents for this registry are as follows:
-
-| Value  | Name                    | Reference |
-|:-------|:------------------------|:----------|
-| 0x0000 | P256_SHA256_AES128GCM   | RFC XXXX  |
-| 0x0001 | X25519_SHA256_AES128GCM | RFC XXXX  |
+~~~
+   CipherSuite MLS_LVL_KEM_AEAD_HASH_SIG = VALUE;
+~~~
 
-[[ Note to RFC Editor: Please replace "XXXX" above with the number assigned to
-this RFC. ]]
+| Component | Contents |
+|:----------|:---------|
+| MLS       | The string "MLS" followed by the major and minor version, e.g. "MLS10" |
+| LVL       | The security level |
+| KEM       | The KEM algorithm used for HPKE in TreeKEM group operations |
+| AEAD      | The AEAD algorithm used for HPKE and message protection |
+| HASH      | The hash algorithm used for HPKE and the MLS KDF |
+| SIG       | The Signature algorithm used for message authentication |
```

Comment:
Please update line 810 to say that ciphersuites are defined by: KEM, AEAD, HASH, SIG.

Reply:
The missing part would be the signature, but the context here is "primitives to be used in group key computations".
I guess we could reference HPKE here.

Reply:
done